src/SNPN/Common/NetworkService.cs

```diff
@@ -154,6 +179,44 @@ public async Task<ResponseResult> SendNotification(QueuedNotificationItem notifi
 			});
 		}
 
+		public async Task<ResponseResult> SendNotificationFCM(QueuedNotificationItem notification)
+		{
+			return await _retryPolicy.ExecuteAsync(async () =>
+			{
+				_logger.Information("SendNotificationFCM {notificationUri}", notification.Uri);
+				var message = new Message()
+				{
+					Data = new Dictionary<string, string>()
+					{
+					["type"] = notification.MatchType.ToString(),
+					["username"] = notification.Post.Author,
+					["title"] = notification.Title,
+					["text"] = notification.Message,
+					["nlsid"] = notification.Post.Id.ToString(),
+					["parentid"] = notification.Post.ParentId.ToString(),
```

Comment:
The Post was needed for a lot of functionality in the ShackBrowse notification, like the parent post ID for muting functionality, the type for various setting enable/disable checks, etc.